src/workflows/orders.ts

```diff
@@ -61,6 +73,58 @@ export async function createOrderWorkflow(
   };
 }
 
+export async function createOrderWorkflowWithSigner({
+  l1Signer,
+  l2Signer,
+  request,
+  ordersApi,
+}: CreateOrderWorkflowWithSignerRequest) {
+  const getSignableOrderResponse = await ordersApi.getSignableOrder({
+    getSignableOrderRequestV3: request,
+  });
+
+  const { signable_message: signableMessage, payload_hash: payloadHash } =
+    getSignableOrderResponse.data;
+
+  // Sign message with L1 credentials
+  const ethSignature = await signRaw(signableMessage, l1Signer);
+
+  // Sign hash with L2 credentials
+  const starkSignature = await l2Signer.signMessage(payloadHash);
```

Comment:
don;t we need `serializeSignature`?

Reply:
we don't, that's have implemented in BaseSigner